docs/get-started/cardano-node/topology.md

```diff
@@ -115,47 +131,94 @@ Additionally, the SIGHUP signal will prompt the system to re-read the block forg
+If a negative value is specified a node will not use ledger peers.
+Ledger peers should be disabled for your block producing node.
+
+Ledger peers are drawn from the ledger based on stake distribution.
+Big ledger peers is a similar notion.
+It is a subset of ledger peers which contains 90% of them with the largest stake.
+
+`PublicRoots` serve as a source of fallback peers, which are used if we are before the configured `useLedgerAfterSlot` slot (please consider using `bootstrapPeers` instead or Genesis).
 
 ### Genesis lite a.k.a Bootstrap Peers
 
-Bootstrap Peers is a pre-Genesis feature that means to provide a way for a node to connect to a trustable set of peers when the its chain falls behind.
+Bootstrap Peers is a pre-Genesis feature that allows a node to connect to a trustable set of peers when its chain falls behind.
 
 Bootstrap peers can be disabled by setting `bootstrapPeers: null`.
 They are enabled by providing a list of addresses.
-By default bootstrap peers are disabled.
+By default, bootstrap peers are disabled.
 
-Trustable peers are composed by the bootstrap peers (see `bootstrapPeers` option in the example topology file above) and the trustable local root peers (see `trustable` option in the example topology file above).
-By default local root peers are not trustable.
+Trustable peers comprise the bootstrap peers (see `bootstrapPeers` option in the example topology file above) and the trustable local root peers (see `trustable` option in the example topology file above).
+By default, local root peers are not trustable.
 
-In order for the node to be able to start and make progress, when bootstrap peers are enabled, the user _must_ provide a trustable source of peers via topology file.
+For the node to be able to start and make progress when bootstrap peers are enabled, the user _must_ provide a trustable source of peers via a topology file.
 This means that the node will only start if either the bootstrap peers list is non-empty or there's a local root group that is trustable.
-Failing to configure the node with trustable peer sources will make it so that the node crashes with an exception.
-*_Please note_* that if the only source of trustable peers is a DNS name, the node might not be able to make progress once in fallback state, if DNS is not providing any addresses.
+Failing to configure the node with trustable peer sources will cause the node to crash with an exception.
+*_Please note_* that if the only source of trustable peers is a DNS name, the node might be unable to make progress once in the fallback state if DNS is not providing any addresses.
 
-With bootstrap peers enabled the node will trace:
+With bootstrap peers enabled, the node will trace the following:
 
 - `TraceLedgerStateJudgmentChanged {TooOld,YoungEnoug}`: If it has changed to any of these states.
```

Comment:
(typo) YoungEnough

Reply:
thanks; typo fixed in https://github.com/cardano-foundation/developer-portal/pull/1387/commits/2f91f45beba476f511e9ef83af1a78995a1ba2c8